src/plugins/dam/sagas/linkAssetsFlow.js

```diff
@@ -83,13 +82,13 @@ export function* failureFlow(reject, error = null, failedCount, expectedCount) {
  * @param config
  */
 export default function* linkAssetsFlow({ pbj, resolve, reject, config }) {
+  yield fork([toast, 'show']);
   yield putResolve(pbj);
   const expectedEvent = config.schemas.assetLinked.getCurie().toString();
   const eventChannel = yield actionChannel(expectedEvent);
-  yield fork([toast, 'show']);
   const result = yield race({
     event: call(waitForMyEvent, eventChannel),
-    timeout: delay(5000),
+    timeout: delay(1000),
```

Comment:
@albertgcsula moving the fork was probably enough.  leave the delay at 5000 for now

Reply:
@gdbrown okay added back